tests/cli_evolution_display_test.py

```
@@ -0,0 +1,143 @@
+"""
+Tests for CLI evolution display formatting.
+
+This module tests that evolution results are displayed properly
+without truncating important content.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)